### PR TITLE
Adds installation of argocd cli to ubuntu

### PR DESCRIPTION
**Pull request by capt2101akash**
Installation of argocd-cli in ubuntu addition.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1256)

### CODE REVIEW

File: content/docs/use-cases/gitops-for-kubeflow.md [hunk 1]
```diff
@@ -38,6 +38,13 @@ kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/$A
 ```shell
 brew install argoproj/tap/argocd
 ```
+2. For Ubuntu
```
Comment (sarahmaddox) [inline content/docs/use-cases/gitops-for-kubeflow.md]: Thanks for adding this info! It looks like the list formatting is a little odd on the page. This was so even before you made the change, and the new info highlights the odd formatting.

I think it should be something like this:

1.  Install Argo CD to the Kubernetes Cluster ...

1. Install the CLI for Argo CD CLI:

    * On macOS: ...
    * On Ubuntu: ...

1. Set extra permissions ...
1. Expose the Argo CD API server...
1. Login using the CLI as admin user ...

I think you'll need to play around with the indentation to make the above format work. It's often also useful to remove the  `shell` style declaration next to the three ticks that start the code blocks, as the `shell` declaration doesn't play well with the list styling.

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1256--competent-brattain-de2d6d.netlify.com/docs/use-cases/gitops-for-kubeflow/

File: content/docs/use-cases/gitops-for-kubeflow.md (hunk 1 above)
Comment (capt2101akash) [reply content/docs/use-cases/gitops-for-kubeflow.md]: Sure, will do that.

**Comment by capt2101akash** [comment]
@sarahmaddox made the requested changes.

File: content/docs/use-cases/gitops-for-kubeflow.md [hunk 2]
```diff
@@ -28,36 +28,35 @@ Follow the [argo cd getting started guide](https://github.com/argoproj/argo-cd/b
...
-brew install argoproj/tap/argocd
-```
-2. For Ubuntu
-```
-wget https://github.com/argoproj/argo-cd/releases/download/v1.2.3/argocd-linux-amd64
-mv argocd-linux-amd64 argocd
-chmod u+x argocd
-export PATH=$PATH:$HOME/path/to/argocd
-```
+2. Install the CLI for Argo CD CLI
```
Comment (sarahmaddox) [inline content/docs/use-cases/gitops-for-kubeflow.md]: It's best to leave the list numbering as `1.` all the way through - that means we don't need to change the numbering if people add or remove steps. The numbers `2.` or `3.` etc have no effect on the formatting.

File: content/docs/use-cases/gitops-for-kubeflow.md [hunk 3]
```diff
@@ -28,36 +28,35 @@ Follow the [argo cd getting started guide](https://github.com/argoproj/argo-cd/b
...
+      * On Ubuntu:
+	```
+           $ wget https://github.com/argoproj/argocd/releases/download/v1.2.3/argocd-linux-amd64
+	   $ mv argocd-linux-amd64 argocd
+	   $ chmod u+x argocd
+	   $ export PATH=$PATH:$HOME/path/to/argocd
+        ```
 
-1. Set extra permissions for Argo CD
+3. Set extra permissions for Argo CD
```
Comment (sarahmaddox) [inline content/docs/use-cases/gitops-for-kubeflow.md]: The list formatting is still broken here. It's not easy! The three ticks that give the code formatting in the above code block need to be at the same indentation level as the code.

Also, indent the other code blocks in the list items below this line to fix the further list formatting.

File: content/docs/use-cases/gitops-for-kubeflow.md (hunk 2 above)
Comment (capt2101akash) [reply content/docs/use-cases/gitops-for-kubeflow.md]: Yeah, I agree. Just thought to make it clearer. :P

**Comment by capt2101akash** [comment]
@sarahmaddox I tried fixing it, can you concur?

File: content/docs/use-cases/gitops-for-kubeflow.md [hunk 4]
```diff
@@ -1,16 +1,18 @@
-+++
```
Comment (sarahmaddox) [inline content/docs/use-cases/gitops-for-kubeflow.md]: The three plus signs `+++` must remain in place. Their removal is causing incorrect rendering at the top of the page. See preview.

Comment (capt2101akash) [reply content/docs/use-cases/gitops-for-kubeflow.md]: Sorry, forgot to add those while committing. Added those in the new commits.

**Comment by capt2101akash** [comment]
@sarahmaddox the preview looks good now. Can you review.

**Comment by sarahmaddox** [comment]
Thanks! There are still a couple of formatting issues which we can fix up later. It's better than before, thanks to your work on the formatting as well as the inclusion of the Ubuntu instructions.

/lgtm
/approve

**Pull request merged**